test/unit/deprecated.js

```diff
@@ -429,8 +429,8 @@ QUnit[ typeof Symbol === "function" && Symbol.toStringTag ? "test" : "skip" ](
 		obj[ Symbol.toStringTag ] = "Function";
 		fn[ Symbol.toStringTag ] = "Object";
 
-		assert.equal( jQuery.isFunction( obj ), false, "function-mimicking object" );
-		assert.equal( jQuery.isFunction( fn ), true, "object-mimicking function" );
+		assert.equal( jQuery.isFunction( obj ), true, "function-mimicking object" );
+		assert.equal( jQuery.isFunction( fn ), false, "object-mimicking function" );
 	}
```

Comment:
I'm confused. Shouldn't a function-mimicking object return `false` when tested against `jQuery.isFunction()` (and `true` for object-mimicking function)? Why the change here?

Reply:
obj is changed to Function a few lines above and typeof gives function for this. And also the new isFunction check. The literal sets it to another type.

Reply:
My PR is just a partly fix as the unit tests sre not correct and the expected behaviour of the new method in general after its deprecation (still do not know what is needed to migrate other things and plugins which rely on these. Personally I hoped there would have been a beta release instead of rushing out with a new one and without testing against big frameworks).

Reply:
@DanielRuf it's always possible to test against master. As you can imagine there are a LOT of major frameworks using jQuery and we can't take on the responsibility of testing every release against them. Does Zurb have its CI set up to test against jQuery betas? If so a beta would have caught this. If not, I don't think it would have made a difference.

Reply:
See package.json. Just (stable) releases are installed by npm.